person.rb

```diff
@@ -4,12 +4,12 @@ class Person < Nameable
   attr_accessor :name, :age
```

Comment:
- [ ] Option 7 does not exit the app. Please ensure that your app is exited after typing the number 7 and clicking on the Enter button.